tests/tests/TestNamesake.cfc

```diff
@@ -0,0 +1,18 @@
+component extends="testbox.system.BaseSpec" {
+	function run(testResults, testBox) {
+		guidesDir = ExpandPath("../guides/en");
+		guides = directoryList(guidesDir, false, "array").map(function(path) {
+			return getFileFromPath(path).replace('.md','');
+		});
+		// temporary: should be removed later
+		index.components = ["application-cfc"];
+
+		describe("JSON Related Tests", function() {
+			it("there should be no guides and function/tag with the same name", function() {
+				for(guide in guides) {
+					expect(not (guide is 'application' OR not fileExists('../data/en/#guide#.json'))).toBeTrue('You cannot create with a guide with the name "#guide#" because there is already a doc with the same name.<br />Tip: Do not hesitate to rename your guide. The title shown in navigation is the title you write into the first line ;-)');
```

Comment:
I'm finding this line confusing because of double negative, you can use toBeFalse() instead of adding the not, and that might help me understand it better... 

Maybe the guide is 'application' part can be moved up into another statement... `if (guide == 'application') { continue; }`

Reply:
Okay, cool. I'll change that!

Reply:
Now it claims to have a doc named "cgi-scope" which is not true...